README.md

```diff
@@ -813,22 +813,28 @@ To upgrade MarkLogic Docker from release 10.x to the latest release, perform fol
 
 Note: In the below example, we are upgrading an initialized MarkLogic host to the latest MarkLogic version supported for Docker.
 
-1. Stop the MarkLogic Docker container.
-Use following command to stop the container:
+1. If you are upgrading to a rootless image, you need to update the ownership of all files and directories under /opt/MarkLogic in the container. Otherwise skip to step 2.
+Use the following two commands to stop the MarkLogic server and update the ownership of the files and directories:
+```
+$ docker exec -it -u root container_id /etc/init.d/MarkLogic stop
 
+$ docker exec -it -u root container_id chown -R 1000:100 /opt/MarkLogic
+```
+2. Stop the MarkLogic Docker container.
+Use following command to stop the container:
 ```
 $ docker stop container_id
 ```
-2. Now, run a MarkLogic Docker container using the latest release of the Docker image. Use the same volume, mounted to the container that was running the older release.
+3. Now, run a MarkLogic Docker container using the latest release of the Docker image. Use the same volume, mounted to the container that was running the older release. Container name should change to avoid conflicts with the older container.
```

Comment:
Technically, the old container can be deleted but the volume preserved if the container name is significant to the system. That way it can be recreated.